src/modules/0.0.5/aws_ec2/entity/instance.ts

```diff
@@ -41,6 +36,12 @@ export class Instance {
   })
   keyPairName: string;
 
+  @Column({
+    type: 'json',
+    nullable: true,
+  })
+  tags: { [key: string]: string }
```

Comment:
Since it is nullable it'd probably be:
```suggestion
  tags?: { [key: string]: string }
```